src/Runtime/jni/jniwrapper.c

```diff
@@ -595,37 +595,12 @@ jobject omtl_native_to_java(
 
     /* Create direct byte buffer Java object from native data buffer.
      *
-     * If jni_owning is true, we take ownership by setting owner flag
-     * to false. This means that when we call omTensorListDestroy
-     * the data buffer will not be freed since it has been given to
-     * the Java direct byte buffer and the Java GC will be responsible
-     * for freeing the data buffer. This way we avoid copying the data
-     * buffer.
-     *
-     * If jni_owning is false, it means the data buffer is not freeable
-     * due to one of the two following cases:
-     *
-     *   - user has malloc-ed the data buffer so the user is
-     *     responsible for freeing it
-     *   - the data buffer is static
-     *
```

Comment:
I wouldn't remove these comments. It serves as a reminder why the output tensor data buffer could have owning=false (due to compiler generating static data).